### DISCUSSION TITLE
Style Guide

### DISCUSSION THREAD

**Discussion by jkpark** [matgip/TastyEstate#60] [Announcements]
기본적으로 Vue의 Style Guide를 따름.
https://vuejs.org/style-guide/

**Comment by jkpark** [comment]
[SFC](https://vuejs.org/guide/scaling-up/sfc.html)의 File name은 PascalCase (MyComponent.vue) not kebab-case (my-component.vue)

**Comment by jkpark** [comment]
# [Component name casing in templates](https://vuejs.org/style-guide/rules-strongly-recommended.html#component-name-casing-in-templates)

Bad
```
<!-- In DOM templates -->
<my-component></my-component>
```

Good
```
<!-- In Single-File Components and string templates -->
<MyComponent/>
```

PascalCase has a few advantages over kebab-case:

Editors can autocomplete component names in templates, because PascalCase is also used in JavaScript.
<MyComponent> is more visually distinct from a single-word HTML element than <my-component>, because there are two character differences (the two capitals), rather than just one (a hyphen).

**Comment by junhong91** [comment]
Private 함수에는 $_를 사용
https://kr.vuejs.org/v2/style-guide/

**Comment by junhong91** [comment]
Vue CSS 영역 이름
- 구분자: 언더바(_)
- 여러단어인 경우: 대시(-)

eg) no-content_title
